README.md

```diff
@@ -86,6 +86,7 @@ Mac OS X Requirements
 xcode-select --install
 brew update
 brew install mercurial
+brew install go
```

Comment:
@codingPerfection I would only ask that you add a note that says to uninstall it after finished with the bootstrap process.

Reply:
@voxxit why uninstall it ?